static/js/multi-level-switch.js

```diff
@@ -12,6 +12,11 @@
 
 /* globals LevelDetail, OnOffDetail, OnOffSwitch, Thing, ThingDetailLayout */
 
+const MULTI_LEVEL_SWITCH_OFF_BAR = 'white';
+const MULTI_LEVEL_SWITCH_OFF_BLANK = '#89b6d6';
+const MULTI_LEVEL_SWITCH_ON_BAR = '#5d9bc7';
+const MULTI_LEVEL_SWITCH_ON_BLANK = 'white';
```

Comment:
It would better if these were CSS classes that are set on the element rather than putting the color codes in JavaScript.

Reply:
Unless there's a particular reason that makes that very difficult..?

Reply:
These are used in the definition of a linear gradient. I could try redesigning the level display to be a div inside of a div which could then be properly styled using classes instead

Reply:
Oh I see, clever. I don't mind if you'd prefer to leave it this way.